app/src/main/res/layout/fragment_news.xml

```diff
@@ -15,9 +15,9 @@
         <TextView
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:text="News"
+            android:text="@string/news_fragment_news"
             android:textStyle="bold"
-            android:textSize="20dp"
+            android:textSize="20sp"
```

Comment:
Comment for all dp to sp changes: Use sp when defining text size, it resizes based on user's preferred text size. 
https://developer.android.com/training/multiscreen/screendensities